rpc/amqp-rpc.go

```diff
@@ -171,7 +173,7 @@ type AmqpRPCServer struct {
 
 // NewAmqpRPCServer creates a new RPC server for the given queue and will begin
 // consuming requests from the queue. To start the server you must call Start().
-func NewAmqpRPCServer(serverQueue string, handler func(*AmqpRPCServer)) (*AmqpRPCServer, error) {
+func NewAmqpRPCServer(serverQueue string, handler func(*AmqpRPCServer), durable, autoAck bool) (*AmqpRPCServer, error) {
```

Comment:
Though I'll note I'm entirely in favor of pulling it out of this PR and discussing the use case/design further somewhere else.